api/stats/rtcstats_objects.h

```diff
@@ -337,6 +337,7 @@ class RTC_EXPORT RTCMediaStreamTrackStats final : public RTCStats {
   RTCStatsMember<double> total_audio_energy;  // Receive-only
   RTCStatsMember<double> echo_return_loss;
   RTCStatsMember<double> echo_return_loss_enhancement;
+  RTCStatsMember<double> echo_likelihood;
```

Comment:
Nit: Can you add "// RingRTC change ..." comments on these too?  And on more below.